sdk/core/Azure.Core/README.md

```diff
@@ -147,20 +147,8 @@ More on handling responses in [response samples](https://github.com/Azure/azure-
 
 ### Consuming Service Methods Returning ```AsyncPageable<T>```
 
-If a service call returns multiple values in pages it would return `Pageable<T>/AsyncPageable<T>` as a result.
-You can iterate over `AsyncPageable` directly or in pages.
-
-```C# Snippet:AsyncPageable
-// call a service method, which returns AsyncPageable<T>
-AsyncPageable<SecretProperties> allSecretProperties = client.GetPropertiesOfSecretsAsync();
-
-await foreach (SecretProperties secretProperties in allSecretProperties)
-{
-    Console.WriteLine(secretProperties.Name);
-}
-```
-
-More on paged responses in [response samples](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Response.md)
+If a service call returns multiple values in pages, it would return `Pageable<T>/AsyncPageable<T>` as a result.
```

Comment:
I think we should keep the sample and remove the section from `Response.md` instead.